README.md

```diff
@@ -33,22 +37,24 @@ Each line in this NDJSON file represents a Digital Collection item, and contains
   "validUntil": 1911,
   "geometry": {
     "type": "Point",
-    "coordinates": [
-      -73.996396,
-      40.732277
-    ]
+    "coordinates": [-73.996396, 40.732277]
   }
 }
 ```
 
 With Space/Time's [field-of-view](https://github.com/nypl-spacetime/field-of-view) module, we can convert this dataset to a GeoJSON file containing field-of-view geometries.
 
+## Running locally
+
+NYPL is no longer maintaining this project, which relies on Gulp, a deprecated version of `node-sass`, and other packages that are only compatible with Python version 2 and Node version 14. To run locally, we suggest using `pyenv` and `nvm` to create a compatible environment.
```

Comment:
You can also suggest running `php -S localhost:8888 -t .` as a local server if the user has php installed on their machine. Slightly easier than having to run this through node or python.